config/settings/local.py

```diff
@@ -19,6 +19,7 @@
 # https://docs.djangoproject.com/en/2.2/ref/settings/#session-cookie-domain
 SESSION_COOKIE_DOMAIN = ".dev.documentcloud.org"
 CSRF_COOKIE_DOMAIN = ".dev.documentcloud.org"
+SESSION_COOKIE_SAMESITE = None
```

Comment:
This is set in `config/settings/base.py` and should not need to be set here also